include/transport/transport.h

```diff
@@ -74,7 +74,7 @@ struct TransportConfig
   const uint32_t time_queue_init_queue_size {1000};     /// Initial queue size to reserve upfront
   const uint32_t time_queue_max_duration {1000};        /// Max duration for the time queue in milliseconds
   const uint32_t time_queue_bucket_interval {1};        /// The bucket interval in milliseconds
-  const uint32_t time_queue_rx_ttl { 250 };             /// Receive queue TTL
+  const uint32_t time_queue_rx_ttl { 500 };             /// Receive queue TTL
```

Comment:
same comment as above, please add time units

Reply:
Will update comments, but units are not fixed where the name can be set to say ms. It's defined by the template variable `duration_t`.